### PR TITLE
ISSD-3945 [PRM] Error messages disappear when adding activity after removing another activity in MDF request FORM.

### PR DESCRIPTION
**Pull request by eduardo-diniz**

### CODE REVIEW

**Comment by ryanschuhler** [comment]
ci:test:sf

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFRequestForm/steps/Activities/Activities.tsx [hunk 1]
```diff
@@ -119,6 +119,7 @@ const Activities = ({
 
 	const onRemove = (index: number) => {
 		setFieldValue(`activities[${index}].removed`, true);
+		arrayHelpers.remove(index);
```
Comment (ryanschuhler) [inline workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFRequestForm/steps/Activities/Activities.tsx:122]: if order is importnat, we should have an extra line break in between. If order is not important then we should alphabetize.

Comment (eduardo-diniz) [reply workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFRequestForm/steps/Activities/Activities.tsx:123]: Done!

**Review by thaynam — APPROVED** [review]

**Comment by thaynam** [comment]
ci:forward:force

**Comment by thaynam** [comment]
ci:forward:force

**Comment by thaynam** [comment]
ci:forward:force

**Pull request closed without merging**